src/styles/components/form-elements/styles.less

```diff
@@ -62,6 +62,14 @@
     bottom: 0.8rem;
```

Comment:
What if we replace this style with something like `top: 2.2rem;` instead of adding two new styles? /cc @mperrotti

Reply:
@nLight You are right, that would be a better solution.